modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.1
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
```

Comment:
@sheriff-rh FYI in case this overlaps with the work you're doing on version numbers?

Reply:
Thanks for checking @abrennan89! This module is not in my 4.14 bump PR. :+1: